tests/suites/main_test.function

```diff
@@ -77,6 +79,12 @@ SUITE_POST_DEP
 /*----------------------------------------------------------------------------*/
 /* Test dispatch code */
 
+#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
+#include <unistd.h>
+#endif
+#include <stdio.h>
```

Comment:
Why is it necessary to have additional includes midway though the source code? `stdio.h` is already included in helpers.function either explicitly or through `platform.h`. If it is not included is because the platform that we are testing on is not meant to use it.

I understand that you might want to keep these includes near the code where they are used. However having the includes spread over multiple files is confusing. Furthermore, they can cause trouble when we are trying to run tests in microcontrollers. Recall that this source code is used by a script to make much larger programs.